### DISCUSSION TITLE
Statamic 5 support

### DISCUSSION THREAD

**Discussion by jolora** [statamic-rad-pack/runway#489] [Feature Requests]
Hi! Yes, I know it's still in beta but just to say Statamic v5 support would be lurrrvly please!

(I'm currently running Statamic 5 beta in an in-development site... I know I shouldn't really but we had already made the decision to build on Laravel 11 without realising that Statamic 4 didn't support v11)

**Comment by duncanmcclean** [comment]
Statamic 5 support is ready to go (see the `master` branch). A new Runway version will be tagged the day Statamic 5 is released.

**Reply by jolora** [reply]
Amazing, thank you so much! Maybe a stupid question but can you see from my composer output here why installing "dev-master" is failing?

```
Package operations: 1 install, 0 updates, 0 removals
  - Syncing statamic-rad-pack/runway (dev-master 5dd67af) into cache
  - Installing statamic-rad-pack/runway (dev-master 5dd67af): Cloning 5dd67af156 from cache
  - Downloading statamic-rad-pack/runway (dist)
    Failed to download
    
    [...]
    
   INFO  Publishing [runway] assets.  
   ERROR  Can't locate path: <vendor/statamic-rad-pack/runway/dist/build/>. 
```

I've set minimum-stability to "dev".

I know... I'm inpatient but I'd like to begin my Runway integration now, before Statamic 5 is officially released :)

**Reply by duncanmcclean** [reply]
It'll be because Runway frontend assets which are only built for tagged released.

In the meantime, you need to run `npm install && npm run build` in the `vendor/statamic-rad-pack/runway` directory after installing via composer.

**Reply by jolora** [reply]
Brilliant, thanks! The build process failed for me so I've just made a forked release. All's working well so far. Thanks again.

**Comment by duncanmcclean** [comment]
Runway [v7.0.0](https://github.com/statamic-rad-pack/runway/releases/tag/v7.0.0) supports Statamic 5.
